packages/iov-bns/src/types.ts

```diff
@@ -177,6 +179,16 @@ export function isSendAction(action: ProposalAction): action is SendAction {
   return action.kind === ActionKind.Send;
 }
 
+export interface SetMsgFeeAction {
+  readonly kind: ActionKind.SetMsgFee;
+  readonly msgPath: string;
+  readonly fee: Fee;
```

Comment:
We could use `readonly fee: Amount;` to simplify types a bit, since we will never need the generic transaction `Fee` type for this price change functionality.

Reply:
I thought about this, but then I thought it might be more confusing than the simplicity was worth.

Reply:
It does reduce the need for validation checks though.

Reply:
I don't have a proper argument for any of the two options. I just think using `Amount` is simpeler. But both ways work.